src/Symfony/Component/Messenger/Command/FailedMessagesShowCommand.php

```diff
@@ -98,12 +102,18 @@ private function listMessages(SymfonyStyle $io, int $max)
                 $errorMessage = $lastRedeliveryStampWithException->getExceptionMessage();
             }
 
-            $rows[] = [
-                $this->getMessageId($envelope),
-                \get_class($envelope->getMessage()),
-                null === $lastRedeliveryStamp ? '' : $lastRedeliveryStamp->getRedeliveredAt()->format('Y-m-d H:i:s'),
-                $errorMessage,
-            ];
+            $currentClassName = str_replace('\\', '', \get_class($envelope->getMessage()));
```

Comment:
What's the purpose of this `str_replace`?

Reply:
It allows to copy/paste quickly a fully qualified class name in the `class-filter` option like `--class-filter=App\Utility\MyClass`

But as the backslash are escaped from the option value, the direct comparison between `\get_class($envelope->getMessage()` and the  `class-filter` option value would have not work without this str_replace.

Reply:
sounds wrong to me to remove all the `\` in FQDN in case someone want copy/paste easily.

I suggest to allows filtering on part of the FQDN (strpos) instead

Reply:
Ok, I actually remove  this str_replace in my last commit to avoid any mistake